.circleci/config.yml

```diff
@@ -168,6 +168,10 @@ workflows:
       - build:
           requires:
             - install
+          filters:
+            branches:
+              ignore:
+              - /docs-preview/
```

Comment:
Nit: can you change this to just `- docs-preview`? In CircleCI these regexes must match the entire string so it shouldn't be a problem, but it doesn't really need to be a regex in the first place.